srcpkgs/n30f/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'n30f'
+pkgname=n30f
+version=2.0
+revision=1
+makedepends="cairo-devel"
+depends="cairo"
+short_desc="Display a PNG in a borderless, transparent, and non-wm-managed window"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/sdhand/n30f"
+distfiles="https://github.com/sdhand/${pkgname}/archive/${version}.tar.gz"
+checksum=7abad075813ee553b3590c0c007799918bbb4fa90c49571bca6505773ece5bf5
+
+do_install() {
```

Comment:
use `post_install()` rather than redefining `do_install()`. remove calls to make below.

Reply:
just use vbin instead